Workbooks/Windows Virtual Desktop/WVD Insights/WVDWorkbookV1.workbook

```diff
@@ -626,7 +626,7 @@
     {
       "type": 1,
       "content": {
-        "json": "The Log Analytics Agent will be [deprecated on August 31, 2024](https://azure.microsoft.com/en-us/updates/were-retiring-the-log-analytics-agent-in-azure-monitor-on-31-august-2024/). To ensure continued support for AVD Insights, please [upgrade to the Azure Monitor Agent](https://learn.microsoft.com/en-us/azure/virtual-desktop/insights?tabs=monitor#session-host-data-settings) and use the updated AVD Insights experience.",
+        "json": "The Log Analytics Agent has been deprecated. To ensure continued support for AVD Insights, please [upgrade to the Azure Monitor Agent](https://learn.microsoft.com/en-us/azure/virtual-desktop/insights?tabs=monitor#session-host-data-settings) and use the updated AVD Insights experience now.",
```

Comment:
Can we keep the link to the deprecation announcement as part of the word `deprecated`? That way the banner has the most context for customers.

Reply:
Updated the banner with deprecation announcement.
<img width="810" alt="image" src="https://github.com/user-attachments/assets/365c6eb8-7022-4428-b2e1-c5c251fe806d">